File: CS/StackedBarTotalLabelSample/Form1.cs
using DevExpress.XtraCharts;
using System;
using System.Windows.Forms;

namespace StackedBarTotalLabelSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            #region #StackedBarTotalLabel
            StackedBarTotalLabel totalLabel = ((XYDiagram)cartesianChart.Diagram).DefaultPane.StackedBarTotalLabel;
            totalLabel.Visible = true;
            totalLabel.ShowConnector = true;
            totalLabel.TextPattern = "Total:\n{TV:F2}";
            #endregion #StackedBarTotalLabel
        }
    }
}

File: VB/StackedBarTotalLabelSample/Form1.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Windows.Forms

Namespace StackedBarTotalLabelSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
'            #Region "#StackedBarTotalLabel"
            Dim totalLabel As StackedBarTotalLabel = CType(cartesianChart.Diagram, XYDiagram).DefaultPane.StackedBarTotalLabel
            totalLabel.Visible = True
            totalLabel.ShowConnector = True
            totalLabel.TextPattern = "Total:" & vbLf & "{TV:F2}"
'            #End Region ' #StackedBarTotalLabel
        End Sub
    End Class
End Namespace
